pytest_selenium/exceptions.py

```diff
@@ -11,4 +11,5 @@ def __init__(self, driver, key, env):
         super(MissingCloudCredentialError, self).__init__(
             '{0} {1} must be set. Try setting the {2} environment '
             'variable, or see the documentation for how to use a '
-            'configuration file.'.format(driver, key, env))
+            'configuration file. For Jenkins credentials, please '
+            'refer to the documentation'.format(driver, key, env))
```

Comment:
I think we can omit the Jenkins mention here. We're already referring to the documentation, which you've updated to cover Jenkins pipelines.